scripts/lcov.sh

```diff
@@ -68,7 +70,7 @@ if [ $# -gt 0 ] && [ "$1" = "--help" ]; then
     exit
 fi
 
-if in_mbedtls_build_dir; then
+if in_mbedtls_repo; then
```

Comment:
Been thinking while writing my comment about backporting: what happens if `scripts/project_name.txt` is absent? It seems to me that it will cause `in_mbedtls_repo` to return false, but as the code is written we'll then just silently assume we're in the TF-PSA-Crypto repo. If we're not, we're likely to get weird errors down the line.

I'm thinking it would probably be better to error out early with a clear reason (`scripts/project_name.txt` not readable). Wdyt?

Reply:
Yes, absolutely a good point. I will implement a safety check here.